tests/Sulu/Component/HttpCache/EventSubscriber/ContentMapperSubscriberTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\HttpCache\EventListener;
+
+use Prophecy\PhpUnit\ProphecyTestCase;
+use Sulu\Component\HttpCache\EventSubscriber\ContentMapperSubscriber;
+use Sulu\Component\Content\Event\ContentNodeDeleteEvent;
+use Sulu\Component\Content\StructureInterface;
+use Sulu\Component\HttpCache\HandlerInterface;
+
+class ContentMapperSubscriberTest extends ProphecyTestCase
+{
+    /**
+     * @var ContentMapperSubscriber
+     */
+    private $subscriber;
+
+    /**
+     * @var HandlerInterface
```

Comment:
That may be, but the point is that this handler implements distinct 3 interfaces.
For this architecture to *make sense* you need an interface which ties them together.

How would you document the type of such an object? The only possiblity is `mixed` which implies that
it could be anything. If the type is `HandlerInterface` then you (or some static analysis) could infer
that it is one of the subtypes of that interface.